index.qmd

```diff
@@ -43,7 +47,28 @@ academically publish, and publicize their software.
 - Collaborate within the Fred Hutch community to build the next generation of
 open source software tools.
 
-<br><br>
+#### Our Contribution
```

Comment:
@seankross did you mean to have these prose changes in this PR?

Reply:
@sckott no, this is a draft because I messed up and probably need to rebase :(